quna/src/main/java/com/ktpt/quna/application/QuestionService.java

```diff
@@ -9,20 +9,23 @@
 import com.ktpt.quna.application.dto.QuestionResponse;
 import com.ktpt.quna.application.exception.NotFoundException;
 import com.ktpt.quna.domain.model.Question;
+import com.ktpt.quna.domain.model.QuestionQueryDao;
 import com.ktpt.quna.domain.model.QuestionRepository;
 
 @Service
 public class QuestionService {
 
     private final QuestionRepository questionRepository;
+    private final QuestionQueryDao questionQueryRepository;
```

Comment:
변수명이 이전 클래스 이름으로 되어있네요

```suggestion
    private final QuestionQueryDao questionQueryDao;
```

IDE의 기능을 통해 이름을 변경하면 필드 이름이나 getter의 이름도 한번에 변경이 되니 잘 활용해보세요

Reply:
오호 미처 발견하지 못한 부분까지, 섬세하시네요!